components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -3426,6 +3426,10 @@ public String getServiceProviderNameByClientId(String clientId, String clientTyp
                                                    String tenantDomain) throws IdentityApplicationManagementException {
         int tenantID = -123;
```

Comment:
Shall we remove this incorrect value assigned here?

Reply:
Fixed in 2ad03b78f04a610e9b0553b8fe72b3709949bc3b by assigning the super tenant Id to preserve intended flow.